mocks/nestedPathsConfig.json

```diff
@@ -149,7 +149,7 @@
             }
         },
         "/with/trailing/slash/": {
-            "get": {
+            "all": {
```

Comment:
I'd keep the old configuration as well, can you add a new path to the conf and specific assertions in the previous integration test?